sites/public/src/components/listings/ListingsList.tsx

```diff
@@ -11,6 +11,7 @@ const ListingsList = (props: ListingsListProps) => {
   return (
     <div>
       <div className="listingsList">{getListings(props.listings)}</div>
+      {getZeroListingsCard(props.listings)}
```

Comment:
IMO this makes more semantic sense as an if statement -- like `if len(props.listings) >0` render `<div className="listingsList">{getListings(props.listings)}</div>
` else `getZeroListingsCard()`" and it also stops the empty listingsList div from rendering

Reply:
:+1:

Reply:
Done.